### DISCUSSION TITLE
allow user to configure cors

### DISCUSSION THREAD

**Discussion by eldemcan** [supabase/supabase#9176] [General]
I am scratching my head tyring to configure basic cors for my http request using RLS and I fail to do so. I think for such fundamental thing supabase should support configuring cors with a simple UI. 

What I tried so far for anonymous roles I added following condition 

```sql
((((current_setting('request.headers'::text, true))::json ->> 'referer'::text) = 'https://www.gigsforpeeps.com/'::text))
```

No luck. so far also there is no origin header is coming with http request according to logs

**Comment by GaryAustin1** [comment]
I believe you are discussing this in Discord with me under a different name (based on on your URL).  One thing there is you started off using a view with a where.  Now you are talking about RLS.  RLS normally won't apply to a view without some special work to change owner roles (I don't use views so don't know those details).

Either try the table directly to see if it is working, or add them to the where in the view.

**Comment by eldemcan** [comment]
@GaryAustin1 thanks for your help in community channel. We could have workarounds for this in RLS level but I think supabase should make easy for us to configure cors. Thats why I opened this discussion hoping that team will pick this as feature request

**Reply by GaryAustin1** [reply]
At least two know from this, https://github.com/supabase/supabase/discussions/8613, and there might be an issue on this too.  But more requests are always better.
